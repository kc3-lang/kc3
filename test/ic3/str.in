" "
"
"
"\0"
"\n"
"\r"
"\s"
"\t"
"\v"
"0"
"9"
"A"
"Z"
"a"
"z"
"À"
"É"
"Ÿ"
"à"
"é"
"ÿ"
"Π"
"꒴"
"𐅀"
"🎳"
"😄"
"🟣"
"🤩"
"""abc"""
"""
abc
"""

"""abc
def"""

"""abc
def
"""

"""
abc
def"""

"""
abc
def
"""
quote "1 + 2 = #{1 + 2}"
"1 + 2 = #{1 + 2}"
