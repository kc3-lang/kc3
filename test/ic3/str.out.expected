" "
"\n"
"\0"
"\n"
"\r"
" "
"\t"
"\v"
"0"
"9"
"A"
"Z"
"a"
"z"
"À"
"É"
"Ÿ"
"à"
"é"
"ÿ"
"Π"
"꒴"
"𐅀"
"🎳"
"😄"
"🟣"
"🤩"
"abc"
"abc"
"abc\ndef"
"abc\ndef\n"
"abc\ndef"
"abc\ndef"
